src/app/app.component.ts

```diff
@@ -0,0 +1,38 @@
+import { AuthService } from '@shared/services/auth.service';
+import { Router, NavigationStart, Event, RouterOutlet } from '@angular/router';
+import { TokenCheckDummyService } from '@shared/services/token-check-dummy.service';
```

Comment:
when you have aliases you can combine your imports into one
`import { AuthService, TokenCheckDummyService } from '@shared/services';`

Reply:
done